department-of-veterans-affairs/abd-vro
# Mcp 2273

Fix BIP test endpoints in swagger UI.

## What was the problem?

Three BIP test endpoints (create a contention, update a contention, and set claim status to RFD) return 500 error.

Associated tickets or Slack threads:

MCP-2273

## How does this fix it?[^1]
Those endpoints used to work. They were broken accidentally by a code style fix in mid December. Revert the accidental change to fix the problem. In addition, I also modify the mock service to return some message to indicate the result from mock service.

## How to test this PR
- Step 1. Build and launch the application as usual.
- Step 2. Test creat-contetion, update-contention, setRFD endpoints. It should not return 500 when proper data is given.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
